charts/stable/resilio-sync/values.yaml

```diff
@@ -38,3 +39,18 @@ securityContext:
     readOnlyRootFilesystem: false
     runAsUser: 0
     runAsGroup: 0
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: http
+              path: /
+            readiness:
+              type: http
+              path: /
+            startup:
+              type: http
+              path: /
```

Comment:
Hmm this is the default already. Probably have to change to tcp unless you can configure basic auth via env vars. In that case we can make http probes work fine

Reply:
Let me switch to TCP then and see